### DISCUSSION TITLE
bicycle access as a quest

### DISCUSSION THREAD

**Discussion by mcliquid** [Helium314/SCEE#539] [General]
I have been following the discussions in Germany and Austria about cycling in forests for some time now. Recently I had the idea of whether it could be an SCEE quest to record access to paths and tracks for bicycles.
Unfortunately, we won't be able to filter in the app whether a path is in the forest or outside it, correct?

That would of course make the quest appear on every path and track no matter where, which would definitely be SPAM in a city or between fields, no matter how many filters you think about.

What do you think of this, since the quest is always hidden anyway, unless you actively activate it?

**Comment by Helium314** [comment]
> Unfortunately, we won't be able to filter in the app whether a path is in the forest or outside it, correct?

This is only possible if the forest and the track are in the same set of tiles where quests are scanned. So for small forests or when surveying near the forest edge it should work, but likely not fully inside a forest.
So whether quests show up or not will be a little hard to predict, because larger download areas (i.e. more tiles scanned at once) are more likely to contain the forest edge, which means the whole forest outline will be downloaded. I'm not sure about the behavior with multipolygons though.

Anyway, I think it would be good to give it a try and see how many tracks/paths it misses.

> What do you think of this, since the quest is always hidden anyway, unless you actively activate it?

I would really like to avoid simply asking the quest for every track or path in Germany and Austria.
If finding paths in forest turns out to be not feasible I may revisit this...

**Comment by mcliquid** [comment]
> Anyway, I think it would be good to give it a try and see how many tracks/paths it misses.

Are there already quests that carry out this kind of filtering that just need to be customised?

**Reply by Helium314** [reply]
There are some that do similar filtering with nodes.
I think address quest, is-this-inside quest, level quest, and possibly some entrance quest.
One of these uses some LatLonRaster or similar for checking what is inside a given area, maybe you could start with this.

Only checking nodes should be sufficient (if it's too complicated for ways), assuming a way through a forest has at least one side inside the forest.

**Comment by mcliquid** [comment]
@ravenfeld Maybe this would be another idea for you to implement? 👍
